x/wasm/types/keys.go

```diff
@@ -75,6 +81,12 @@ func GetContractByCodeIDSecondaryIndexPrefix(codeID uint64) []byte {
 	return r
 }
 
+// GetContractByCreatorThirdIndexKey returns the key for the second index: `<prefix><creatorAddress><contractAddr>`
```

Comment:
As we can have a mix of 20 and 32 byte (contract) addresses, it would make sense to prefix the first address so that there is no overlap. The SDK has `address.MustLengthPrefix()` that can be used